### DISCUSSION TITLE
영어를 지원했으면 좋겠어요

### DISCUSSION THREAD

**Discussion by da-in** [Yangjaecheon-Otter-Guardians/simple-thumbnail#130] [Ideas]
### 주제

Features

### Body

🌏 simple thumbnail의 글로벌 진출을 위해 영어도 지원했으면 좋겠습니다.

### BDD TASK

```bash
- [ ] 한국어 / 영어 선택 토글 버튼이 화면 상단에 보인다.
- [ ] 토글한 언어에 따라 화면의 언어들이 바뀐다.
```

**Comment by caseBread** [comment]
오 굉장히 좋은 기능인 것 같습니다. 다소 직관적으로 사용하는 서비스이기때문에 복잡한 번역도 요구되지 않을것 같습니다.

아래문서 참고해서 공부좀 해보겠습니다. 🙂
[mdn - i18n](https://developer.mozilla.org/ko/docs/Glossary/Internationalization)

http 헤더에 [accept-language](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language) 값을 통해 분기처리도 가능할 것 같습니다. 기회가된다면 언어선택 토글대신 자동화되게끔 도전해보고싶네요.

**Reply by da-in** [reply]
오오 첫 진입시 자동으로 언어를 지정해주고, 그 이후에 토글도 가능하다면 매우 편할 것 같습니다👍🏻
